apps/settings/utils/ldap.py

```diff
@@ -665,7 +687,7 @@ def test_config(self):
     # test login
 
     def _test_before_login_check(self, username, password):
-        from settings.ws import CACHE_KEY_LDAP_TEST_CONFIG_TASK_STATUS, TASK_STATUS_IS_OVER
+        from settings.ws import CACHE_KEY_LDAP_TEST_CONFIG_TASK_STATUS
         if not cache.get(CACHE_KEY_LDAP_TEST_CONFIG_TASK_STATUS):
             self.test_config()
 
```

Comment:
None of the given code appears to be invalid, but it doesn't seem like there's an exact match with my current knowledge cutoff of December 18th, 2021. My abilities might have evolved since then!